activerecord/lib/active_record/relation/predicate_builder/basic_object_handler.rb

```diff
@@ -8,8 +8,8 @@ def initialize(predicate_builder)
       end
 
       def call(attribute, value)
-        bind = predicate_builder.build_bind_attribute(attribute.name, value)
-        attribute.eq(bind)
+        bind = predicate_builder.build_query_attribute(attribute.name, value)
+        bind ? attribute.eq(bind) : "1=0"
```

Comment:
I think `attribute.in([])` would be more consistent with the range handler, as a more abstract false.

Reply:
Yeah, changed to `attribute.in([])`.